src/main/java/com/ericsson/ei/controller/SubscriptionControllerImpl.java

```diff
@@ -118,9 +118,9 @@
                 Subscription subscription = subscriptionService.getSubscription(subscriptionName);
                 subscription.setPassword("");
                 foundSubscriptionList.add(subscription);
-                LOG.debug("Subscription is fetched: " + subscriptionName);
+                LOG.debug("Subscription could not be fetched: " + subscriptionName);
```

Comment:
Shouldn't it be "Subscription fetched successfully" since it is in successful state here.

Reply:
Done